upup/models/cloudup/resources/addons/kube-dns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -53,11 +53,13 @@ spec:
     spec:
       containers:
       - name: autoscaler
-        image: k8s.gcr.io/cluster-proportional-autoscaler-{{Arch}}:1.4.0
+        image: k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.4.0
         resources:
             requests:
                 cpu: "20m"
                 memory: "10Mi"
+            limits:
+                memory: "10Mi"
```

Comment:
Any reason this memory limit is being added for kubedns but not coredns?

Reply:
I'll move this and the kube-dns bump to a different PR, I think.